docs/DEVELOPMENT.md

```diff
@@ -35,6 +34,32 @@ npx tauri dev
 
 Automatic recompiling on save is enabled for both the Rust and the Typescript/Vue code.
 
+If you want to build, do all of the above (except tauri dev) and:
+
+```sh
+cd ../src-tauri
+```
+
+```sh
+npm run tauri build
+```
+
+This will build the executable and bundles, such as AppImage or .deb.
+
+To build just the executable, edit [tauri.conf.json](https://github.com/R2NorthstarTools/FlightCore/blob/c102fdcf8ef00e3c5c46fb9ded37e09b7e92e483/src-tauri/tauri.conf.json#L18) in the same folder.
```

Comment:
`npx tauri build ` will always produce a bundle (`msi`, `deb`, AppImage), there isn't really a way to get just the executable (well, sorta).

Reply:
So editing the config is not the solution?

Reply:
Well, editing the config, allows you to build the bundle without having to give a private key (at the cost of not having an self-update mechanism anymore).

Reply:
49 line in the commit isn't related to the updater.

Reply:
The 49th line in the commit isn't related to the updater.
I don't know what I have to change here.
I've tested this plenty of times, and [bundle active false](https://github.com/R2NorthstarTools/FlightCore/blob/c102fdcf8ef00e3c5c46fb9ded37e09b7e92e483/src-tauri/tauri.conf.json#L18) always disabled the bundle for me.
The only change I can think of is to mention it this way:
```
    "bundle": {
      "active": true,
```
in case of an update to tauri.conf.json.

Reply:
Oh, never tried changing that xD

Never looked at what would happen when disabling that ^^